src/pages/Landing.tsx

```diff
@@ -18,52 +11,13 @@ import { SignInTrezorForm } from "components/SignIn/SignInTrezorForm";
 import { WalletButton } from "components/WalletButton";
 
 import { wallets } from "constants/wallets";
-import { pageInsetStyle } from "constants/styles";
 import { resetAlbedoAction } from "ducks/wallet/albedo";
 import { resetLedgerAction } from "ducks/wallet/ledger";
 import { resetFreighterAction } from "ducks/wallet/freighter";
 import { resetTrezorAction } from "ducks/wallet/trezor";
 import { logEvent } from "helpers/tracking";
 import { ModalType } from "types/types.d";
 
-const WrapperEl = styled.div`
-  ${pageInsetStyle};
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 4rem;
-`;
-
-const WalletButtonsWrapperEl = styled.div`
-  width: 100%;
-  max-width: 590px;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  padding: 3rem 0;
-
-  & > div {
-    margin-bottom: 1.5rem;
-    width: 100%;
-
-    @media (min-width: 700px) {
-      width: calc(50% - 1.75rem);
-    }
-  }
-`;
-
-const ButtonsWrapperEl = styled.div`
-  padding: 1.5rem 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-
-  & > button {
-    margin-bottom: 0.5rem;
-  }
-`;
```

Comment:
What's the reasoning for replacing these elements with css? Looks like we're getting into kind of a weird state in this repo where some things are done with styled-components and others with css

Reply:
I'm trying to match components/styles to SDS as much as possible to make it easier to move them over. Ideally, most of these will come from the SDS eventually. 🤞 Some of them could still be styled-components in the end if it's better that way.

I know we don't have a plan on how to add SDS to current projects, but the idea for AV is to have it fully support dark mode when this feature is done. I thought it would make things easier to use CSS instead of styled-components. Open to other strategies!

Reply:
Ah, got it - these will eventually move to SDS. That makes sense.

Yeah, at some point we should figure out what integrating SDS into all the projects looks like. The main thing I want to avoid is sometimes using scss files and sometimes using styled-components

Reply:
💯 agreed. This project can be a test run of how it works. I'll try to keep notes and what could be improved.